lib/src/screen/dice/dice_screen.dart

```diff
@@ -30,35 +28,19 @@ class _DiceScreenState extends State<DiceScreen> {
           ),
         ),
         body: Center(
-          child: StreamBuilder(
-            stream: widget.diceBloc.diceRoll,
-            initialData: RollEvent(
-              stateType: RollStateType.initial,
-            ),
-            builder: (BuildContext context, snapshot) {
-              return Column(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: <Widget>[
-                  Text(
-                    'Roll a dice:',
-                  ),
-                  Container(
-                    height: Dimension.diceContainerSize,
-                    child: RollWidget(
-                      rollEvent: (snapshot.hasData)
-                          ? snapshot.data
-                          : RollEvent(
-                              stateType: RollStateType.loading,
-                            ),
-                    ),
-                  ),
-                ],
-              );
-            },
-          ),
-        ),
-        floatingActionButton: OnPressWidget(
-          diceBloc: widget.diceBloc,
+            child: ListView.builder(
```

Comment:
Using a listview, is probably overkill for this particular scenario

Reply:
I change the ListView with a Wrap and a SingleChildScrollView